### PR TITLE
Microflix Project

### PR DESCRIPTION
**Pull request by Aquinasmoriasi**
- Added a function to fetch movies from an external API and render the specific information from the fetched items to the page.
- Added styling to the rendered page.
- Added function for posting new likes to the involvement API.
- Add functionality to count and display the number of likes and comments.
- Added function to allow users to post comments to the API through the form input.
- Added a popup to display comments when the user clicks on the comments button.
- Added tests for the functions counting the comments and likes.

### CODE REVIEW

File: src/modules/comments.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  constructor(item_id, username, comment) {
+    this.item_id = item_id;
+    this.username = username;
+    this.comment = comment;
+  }
+}
+
+const postComment = async (comment) => {
+  const posts = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/pRSMj4Sa1SZFpBHfoj5I/comments', {
+    method: 'POST',
```
Comment (KDlamini) [inline src/modules/comments.js:13]: The functionality of adding a comment is not very consistent. Sometimes it works and sometimes it doesn't.

- The app actually behaves in an unusual manner, when I start by liking any tv show, then attempt to add a comment it works. But if I were to refresh the page, then try to add comments again, it doesn't work anymore.
- Kindly make sure that a comment can be added to all the TV Shows shows at all times. In general, adding comments works, but for some reason sometimes it decides not to work.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @Aquinasmoriasi, @businesskaniki 🙋‍♀️,  

Excellent work! 💯 
You have done a great job creating a working app API integration.
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Good API integration
✔️  Webpack configured correctly
✔️  ES6 and async/await used properly
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Team, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  [ ] Kindly deploy your project to netlify or Heroku, because that is only going to need to show your work in the future.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by businesskaniki** [comment]
thank you Aquinas for your contribution

**Comment by Aquinasmoriasi** [comment]
We make a great team. 👏🏽

**Pull request merged**